src/IO/ADIOS/ADIOS2IOHandler.cpp

```diff
@@ -2995,6 +2995,19 @@ namespace detail
                 m_IO.SetParameter("Profile", "Off");
             }
         }
+        if (notYetConfigured("AsyncWrite"))
+        {
+            if (1 == auxiliary::getEnvNum("OPENPMD_ADIOS2_ASYNC_WRITE", 1) &&
```

Comment:
```suggestion
            if (1 == auxiliary::getEnvNum("OPENPMD_ADIOS2_ASYNC_WRITE", 0) &&
```

Your change would activate async write by default. This probably explains the failing CI run, too.